tools/quickstart/quickstart/validator_account.py

```diff
@@ -171,3 +175,38 @@ def get_author_address(base_dir) -> str:
             return author_address_file.read()
     except FileNotFoundError:
         return get_validator_address(base_dir)
+
+
+def handle_legacy_validator_key_file(base_dir, chain_dir):
+    legacy_key_file_paths = legacy_validator_key_paths(base_dir, chain_dir)
+    key_file_paths = legacy_key_file_paths.copy()
+    if validator_key_file_exists(base_dir, chain_dir):
```

Comment:
Please do this check at the beginning of the function and if the validator key file exists just return.
That way you also don't have to copy the legacy_key_file_paths.

Reply:
I can do that, but then if there is a validator key file in the format `account.json` and one in the format `UTC-*` then we will arbitrarily take the `account.json` one into account and ignore the other one. The idea there was to raise an error if this occurs.
Please confirm you still want me to change it and ignore the case where there are two keys.

Reply:
I guess that's also what Bernd wants, so I'll do that.

Reply:
That is fine, at least I fail to see a problem and Bernd has requested the same below. So, yes, please do it.